### PR TITLE
Updated 06. environment.md in first week of mlzoomcamp 

### PR DESCRIPTION
**Pull request by MemoonaTahira**
1. I created a summary table with markdown links to the notes below to better determine what someone can use for ML. 

I kept order from learning locally (windows, windows wsl, macOS and ubuntu) to online via kaggle/colab and cloud services.

I wanted to add Heroku as a deployment service only, unlike a full fledged cloud provider with training environments, but I don't know much about it. It can be added after last row in the summary table with notes.

2. Added notes to 08-Linear Algebra for week 1

3. Added notes to week 2 for overview of work done so far

### CODE REVIEW

File: course-zoomcamp/01-intro/06-environment.md [hunk 1]
```diff
@@ -41,7 +45,7 @@ Anaconda is recommended.
 
 It is a good idea to set up a dedicated environment for the course 
 
-In your terminal, run this command to create the environment
+In your terminal (e.g bash for Ubuntu, anaconda command prompt for windows), run this command to create the environment
```
Comment (alexeygrigorev) [inline course-zoomcamp/01-intro/06-environment.md:48]: I'd actually recommend using Windows Terminal, not anaconda prompt 

https://apps.microsoft.com/store/detail/windows-terminal/9N0DX20HK701

File: course-zoomcamp/01-intro/06-environment.md [hunk 2]
```diff
@@ -132,6 +118,32 @@ Steps for Google Colab are same as that for Kaggle, except for some changes in S
    https://colab.research.google.com/github/alexeygrigorev/mlbookcamp-code/blob/master/chapter-02-car-price/02-carprice.ipynb
 
 
+## Cloud
```
Comment (alexeygrigorev) [inline course-zoomcamp/01-intro/06-environment.md:121]: Can you please move it back?

**Comment by alexeygrigorev** [comment]
Can you maybe create separate PR for the notes and for linear algebra? I'll accept them - and for the rest we need a few iterations

File: course-zoomcamp/01-intro/06-environment.md (hunk 1 above)
Comment (MemoonaTahira) [reply course-zoomcamp/01-intro/06-environment.md:48]: Okay, this terminal looks great.

File: course-zoomcamp/01-intro/06-environment.md (hunk 2 above)
Comment (MemoonaTahira) [reply course-zoomcamp/01-intro/06-environment.md:121]: okay

**Comment by alexeygrigorev** [comment]
Can you please open another PR? This one has conflicts and too many things at the same time. Better to create one for each separate thing

**Pull request closed without merging**